ram/wram.asm

```diff
@@ -2212,6 +2213,7 @@ wMapStatus:: db
 wMapEventStatus:: db
 
 wScriptFlags::
+; bit 2: running script
 ; bit 3: run deferred script
```

Comment:
Once we define constants for the bits.. we don't really need the comments here anymore imo.

Reply:
Maybe not, but we already have a lot of bit comments when constants already exist, so I didn't want to delete them all in this PR.